src/qibo/backends/_clifford_operations.py

```diff
@@ -0,0 +1,471 @@
+from functools import reduce
+
+import numpy as np
+from scipy import sparse
+
+name = "numpy"
+
+np = np
```

Comment:
This is not needed :)

In this scope, `np` is already bound to the `numpy` module, by the `import numpy as np` statement ;)